src/js/components/modals/AppModalComponent.jsx

```diff
@@ -49,7 +49,8 @@ var AppModalComponent = React.createClass({
     return {
       fields: AppFormStore.fields,
       errorIndices: {},
-      responseErrorMessages: {}
+      responseErrorMessages: {},
+      force: false
```

Comment:
I'm not sure about changing the text of the button, as I mentioned in mesosphere/marathon#2298 . @pierlo-upitup what do you think?